raaghu-elements/src/rds-input/rds-input.tsx

```diff
@@ -253,13 +268,22 @@ const RdsInput = React.forwardRef<HTMLInputElement, RdsInputProps>((props, ref)
                         </div>
                     </Tooltip>
                 )}
-                {props.required && (!props.validationMsg) && (
-                    <div className="form-control-feedback">
-                        {props.required && props.value == "" && hasError && isTouch && (
-                            <span className="text-danger">{props.label} {t("is required") || ""} </span>
-                        )}
-                    </div>
-                )}
+                <div className="form-control-feedback d-block text-end">
```

Comment:
@harshitsaini-waiin we don't pass validation message directly from element as we are handling error message with respect to label

Reply:
@Ketan-Khalate resolved above comments.
Handled validations from component and removed validations which were passed directly from element.